src/ServerConnection/SubscriptionEventMessage.ts

```diff
@@ -1,179 +1,159 @@
 import type { CommonMessage } from './CommonMessage.js';
 import type { SubscriptionEvent } from './SubscriptionEvent.js';
-
 type DebugLogSubscriptionEventMessage = CommonMessage<'Subscription'> & {
-  eventType: 'DebugLog';
-  data: unknown;
+    eventType: 'DebugLog';
+    data: unknown;
 };
-
 type ErrorLogSubscriptionEventMessage = CommonMessage<'Subscription'> & {
-  eventType: 'ErrorLog';
-  data: unknown;
+    eventType: 'ErrorLog';
+    data: unknown;
 };
-
 type FatalLogSubscriptionEventMessage = CommonMessage<'Subscription'> & {
-  eventType: 'FatalLog';
-  data: unknown;
+    eventType: 'FatalLog';
+    data: unknown;
 };
-
 type InfoLogSubscriptionEventMessage = CommonMessage<'Subscription'> & {
-  eventType: 'InfoLog';
-  data: unknown;
+    eventType: 'InfoLog';
+    data: unknown;
+};
+type InventoryChangedSubscriptionEventMessage = CommonMessage<'Subscription'> & {
+    eventType: 'InventoryChanged';
+    data: {
+        user: {
+           id: number,
+           username: string
+        },
+        itemname: string,
+        quantity: number,
+        itemhash: string,
+        material?: string,
+        savestring: string,
+        changetype: string,
```

Comment:
use the union model or whatever TS calls it, like below.

source: 'Unknown' | 'Impact' | 'FallDamage';